qiita_db/metadata_template.py

```diff
@@ -2228,20 +2229,32 @@ def load_template_to_dataframe(fn, strip_whitespace=True):
     |             longitude |        float |
     +-----------------------+--------------+
     """
+    # Load in file lines
+    holdfile = None
+    if isinstance(fn, (str, unicode)):
+        # filepath passed
+        with open(fn) as f:
+            holdfile = f.readlines()
+    else:
+        # open file passed
+        holdfile = fn.readlines()
+    if len(holdfile) == 0:
```

Comment:
Can just be `if not holdfile` since it will either be a list of lines (non-empty file was read), an empty list (empty file was passed), or an exception would have been raised somewhere between lines 2234 and 2240.